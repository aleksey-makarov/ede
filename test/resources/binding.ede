{}
---
{% let x = 123 %}
  {{ x }}

  {% let y = 321 %}
    {{ x }} + {{ y }} = {{ x + y }}
  {% endlet %}

{% endlet %}
