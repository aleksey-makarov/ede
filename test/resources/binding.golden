  123

    123 + 321 = 444

